### PR TITLE
fixed issue #510

### LINKED ISSUE #510: Bug Report: _Current UX reference Page doc seems out of flow and need a more stable structure
**Describe the bug**
The current [Talawa UI/UX reference page](https://docs.talawa.io/docs/design/ux/ux-talawa) seems out of structure, the reader may find this hard to go through as the sections are intermingled the flow of the page is not quite consistent

**Expected behavior**

https://user-images.githubusercontent.com/72331432/225412487-a5d6d1e9-1573-41b7-8def-a81ce7f765e0.mp4

I have tried to structure the page by making the particular heading bold and divided each section properly

**Actual behavior**
Overview _ Talawa Project Documentation — Mozilla Firefox 3_16_2023 12_17_31 AM

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

I would like to solve this issue

### PR DESCRIPTION
**Pull request by jmalicha**
**What kind of change does this PR introduce?**
- Improving the flow of the reference page by making it uniform

Issue 
- Fixes  #510 

**Snapshots/Videos:**
**Here is the screenshot and a video after correction**
t1

https://user-images.githubusercontent.com/51859049/226140801-6edb840f-59e2-499d-b8a0-42eb065c4bdb.mp4

**Before correction**
Here is the actual behavior of the page
t2

**Does this PR introduce a breaking change?**
No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by jmalicha** [comment]
Hello @Cioppolo14 kindly review what I have done. Thanks

**Review by Cioppolo14 — CHANGES_REQUESTED** [review]
@jmalisha Could you please read through the text and correct legacy grammar and spelling typos?

**Comment by jmalicha** [comment]
@Cioppolo14 Yeah sure.

**Comment by jmalicha** [comment]
@Cioppolo14 I have corrected the typos, please have a look at it. Thanks

File: docs/design/talawa-branding.md [hunk 1]
```diff
@@ -46,7 +47,7 @@ This section describes the significance of the logo.
 
 ![Small Logo](/img/logos/talawa-logo-200x200.png)
 
-1. The logo is an abstract representation of a small human being. 'Talawa' in the Jamaican dialect means 'strong', and the logo symbolizes the a well known saying on the island 'Little, but Talawa'.
+1. The logo is an abstract representation of a small human being. 'Talawa' in the Jamaican dialect means 'strong', and the logo symbolizes the a well-known saying on the island 'Little, but Talawa'.
```
Comment (Cioppolo14) [inline docs/design/talawa-branding.md]: Please remove the "a" after symbolizes

File: docs/design/ux/talawa.md [hunk 2]
```diff
@@ -7,44 +7,69 @@ title: Talawa UI/UX
...
+On all screens
+
+#### Colors:
+
+1.  Colors must be consistent:
+
+    - With the color palette.
+    - In their use in design elements
+
+2.  The colors used for must imply the same functionality. For example, a depressed button cannot
```
Comment (Cioppolo14) [inline docs/design/ux/talawa.md]: Please remove "for" after the word used

File: docs/design/ux/talawa.md [hunk 3]
```diff
@@ -7,44 +7,69 @@ title: Talawa UI/UX
...
+    1.  Main Elements:
+
+    - Should be white with gray/black borders, or black with gray/white borders (dark mode) whenever possible
+
+    2.  Design Highlights:
+
+    - Shades of gray should be preferred choice for design highlight elements
+    - Other color palette selections can be considered as a secondary option
+
+5.  Unselected design elements should be grayed out in a consistent gray and no other color
```
Comment (Cioppolo14) [inline docs/design/ux/talawa.md:35]: Is the 5 and 6 the current number for these two items?

File: docs/design/ux/talawa.md [hunk 4]
```diff
@@ -7,44 +7,69 @@ title: Talawa UI/UX
...
-1. Images:
-    1. Images must only have square edges. This reduces the risk of losing content due to cropping.
-1. Positioning:
-    1. All design elements must be a consistent distance from the edge of the screen. This includes:
-        1. Icons
-        1. Text
-        1. Dialog boxes
-    1. If a design element stretches across the screen then it must be equidistant from both edges of the screen.
-        1. For example: A design element consisting of an icon grouped to the left of a text dialog box should ensure the icon is the same distance from the left of the screen as the right most edge of the dialog box is from the right side of the screen.
+### Style Guide
```
Comment (Cioppolo14) [inline docs/design/ux/talawa.md:10]: Change "on all screens" to "The guide below applies to all screen types."

File: docs/developers/talawa-api/mutations.md [hunk 5]
```diff
@@ -4,8 +4,10 @@ title: Mutations
...
 
 ## Mutations According to Roles
-This doucument outlines our mutations and to which type of user each is constrained
-### Type Of Users in Talawa 
+
+This document outlines our mutations and to which type of user each is constrained
+
+### Type Of Users in Talawa
 
 There are mainly 3 types of User types in talawa -
```
Comment (Cioppolo14) [inline docs/developers/talawa-api/mutations.md]: Uncapitalize User and capitalize talawa please

File: docs/developers/talawa-api/mutations.md [hunk 6]
```diff
@@ -16,14 +18,16 @@ There are mainly 3 types of User types in talawa -
 3. `USER`
 
 ### Mutations Permitted by Roles.
+
 The limits of each role's use of mutations are stated below:
+
 #### `SUPERADMIN` -
 
-These are the Mutations exclusive to the `SUPERADMIN` type. Please note that a few of these mutations also have a few extra criteria with who can access them .For eg. except `createOrganization` mutation all other require that SUPERADMIN should also be the creator of the orgnanization.
+These are the Mutations exclusive to the `SUPERADMIN` type. Please note that a few of these mutations also have a few extra criteria with who can access them. For example, except `createOrganization` mutation all other require that SUPERADMIN should also be the creator of the orgnanization.
```
Comment (Cioppolo14) [inline docs/developers/talawa-api/mutations.md]: Please change this paragraph to the following:
"Below are the Mutations exclusive to the `SUPERADMIN` type. Please note, a few of these mutations have a few extra criteria regarding user access. For example, except for the `createOrganization` mutation, all other mutations require the SUPERADMIN to also be the creator of the orgnanization."

File: docs/developers/talawa-api/mutations.md [hunk 7]
```diff
@@ -16,14 +18,16 @@ There are mainly 3 types of User types in talawa -
...
 
 1. `blockPluginCreationBySuperadmin`
 2. `createAdmin`
-3. `createOrganization` - When a superadmin creates and organization , he/she will aso be one of the `admins` and `creator` of that Organization. Therefore, the Mutations in the next Sections also applies to Super Admin.  
+3. `createOrganization` - When a superadmin creates and organization, he/she will also be one of the `admins` and `creator` of that Organization. Therefore, the Mutations in the next Sections also applies to Super Admin.
```
Comment (Cioppolo14) [inline docs/developers/talawa-api/mutations.md]: remove the s at the end of applies

File: docs/functionalities/core-functionalities.md [hunk 8]
```diff
@@ -3,23 +3,25 @@ id: core-functionalities
...
+> The current Talawa app is functional but does not meet many best practices. We need to refactor the code to be internationally acceptable.
 
 > Before we can add any new functionality. The refactored app needs to meet these basic requirements on which the original application was based.
 
 ## User Registration
+
 ### Description
-This is the ability to register in the app with an organisation. Once the user is successfully registered as a member of an organisation they are then able to interact with, app events and other content related to the organisation.
+
+This is the ability to register in the app with an organization. Once the user is successfully registered as a member of an organization they are then able to interact with, app events and other content related to the organization.
```
Comment (Cioppolo14) [inline docs/functionalities/core-functionalities.md]: Please change to: 
This is the ability to register in the app with an organization. Once the user is successfully registered as a member of an organization, they are then able to interact with app events and other content related to the organization.

**Review by Cioppolo14 — CHANGES_REQUESTED** [review]
Thank you for looking at all this!

File: docs/design/ux/talawa.md (hunk 4 above)
Comment (jmalicha) [reply docs/design/ux/talawa.md:10]: On it

File: docs/design/ux/talawa.md (hunk 2 above)
Comment (jmalicha) [reply docs/design/ux/talawa.md]: Done

File: docs/design/ux/talawa.md (hunk 3 above)
Comment (jmalicha) [reply docs/design/ux/talawa.md:35]: Yes. I double-checked unless there is something I am missing. Will have a look at it again

File: docs/design/ux/talawa.md (hunk 4 above)
Comment (jmalicha) [reply docs/design/ux/talawa.md:10]: Done

File: docs/developers/talawa-api/mutations.md (hunk 5 above)
Comment (jmalicha) [reply docs/developers/talawa-api/mutations.md]: Done

File: docs/developers/talawa-api/mutations.md (hunk 6 above)
Comment (jmalicha) [reply docs/developers/talawa-api/mutations.md]: Done

File: docs/developers/talawa-api/mutations.md (hunk 7 above)
Comment (jmalicha) [reply docs/developers/talawa-api/mutations.md]: that means I should put "apply" right?

File: docs/functionalities/core-functionalities.md (hunk 8 above)
Comment (jmalicha) [reply docs/functionalities/core-functionalities.md]: Done

**Comment by jmalicha** [comment]
Done with all requested changes

File: docs/developers/talawa-api/mutations.md (hunk 7 above)
Comment (Cioppolo14) [reply docs/developers/talawa-api/mutations.md]: @jmalisha That's correct, thanks for catching that.

**Review by Cioppolo14 — APPROVED** [review]

**Pull request merged**